modules/melee_mode.lua

```diff
@@ -143,10 +143,10 @@ local function player_inventory_changed(player_index, inv_id, name)
    local inv = player.get_inventory(inv_id)
    local moved = move_to_main(player, inv, player.get_main_inventory())
    if #moved > 0 then
-      player.print('In melee mode, moved ' .. moved_to_string(moved) .. ' to main inventory')
+      player.print({ "melee_mode.move_to_main_inventory", moved_to_string(moved) })
    end
    if not inv or not inv.is_empty() then
-      player.print('WARNING: in melee mode, unable to empty ' .. name .. ' to main inventory')
+       player.print({ "melee_mode.move_to_main_inventory_failed", moved_to_string(moved) })
```

Comment:
You need to include `name` which is either `ammo` or `guns` depending on which event is calling the function